sdk/python/feast/client.py

```diff
@@ -442,6 +443,46 @@ def archive_project(self, project):
         if self._project == project:
             self._project = opt().PROJECT
 
+    def apply(self, objects: Union[List[Union[Entity, FeatureTable]], Entity, FeatureTable], project: str = None):
+        """
+        Idempotently registers entities and feature tables with Feast Core. Either a single
+        entity or feature table or a list can be provided.
+
+        Args:
+            objects: List of entities and/or feature tables that will be registered
+
+        Examples:
+            >>> from feast import Client
+            >>> from feast.entity import Entity
+            >>> from feast.value_type import ValueType
+            >>>
+            >>> feast_client = Client(core_url="localhost:6565")
+            >>> entity = Entity(
+            >>>     name="driver_entity",
+            >>>     description="Driver entity for car rides",
+            >>>     value_type=ValueType.STRING,
+            >>>     labels={
+            >>>         "key": "val"
+            >>>     }
+            >>> )
+            >>> feast_client.apply(entity)
+        """
+
+        if project is None:
+            project = self.project
+
+        if not isinstance(objects, list):
+            objects = [objects]
+        for obj in objects:
+            if isinstance(obj, Entity):
+                self._apply_entity(project, obj)  # type: ignore
+                continue
+            elif isinstance(obj, FeatureTable):
```

Comment:
Is there a reason to use `continue` if you are using an `elif`?

I would expect either 
```
if a:
  do
  continue

if b:
  do 
  continue

raise
```
or
```
if a:
  do
elif b:
  do
else:
 raise
```